sonarqube/tox.ini

```diff
@@ -22,6 +24,12 @@ deps =
 commands =
     pytest -v {posargs}
 setenv =
-    LEGACY_DOCKER_COMPOSE = true
+    COMPOSE_FILE=docker-compose.yaml
```

Comment:
Do we have another docker-compose file?

Reply:
Agreed. Do we have something on the python side to wait for the container to be healthy?